src/main/java/com/origamisoftware/teach/agile/junit/SubscriptionPeriod.java

```diff
@@ -40,20 +40,33 @@ public Date getEndDate() {
     }
 
     /**
-     * @return the total Days in the subscription
+     * @return the total Days in the subscription, to start 0
      */
     public int getTotalDays() {
-        // todo implement method
-        return 0;
+        /** todo implement method */
+        long end = endDate.getTime();
+        long start = startDate.getTime();
+        long diff = Math.abs(endDate.getTime() - startDate.getTime());
+        diff = diff / 1000;
+        diff = diff / 86400;
+     int tmptotalDays = (int) diff;
+        return tmptotalDays;
```

Comment:
looks good. in this method you don't need the tmptotalDays variable just return (Int)diff;
Also use camel case when naming variables - that the java standard 
Also use whole words when naming variables - code is easy to ready.